src/css/MoviesContainer.css

```diff
@@ -10,5 +10,36 @@
 	grid-auto-rows: auto;
   /* border: solid 2px yellow; */
   place-items: center;
+}
 
+@media screen and (max-width: 1430px) {
+.MoviesContainer {
+	overflow-y: scroll;
+	display: grid;
+	  grid-template-columns: repeat(3, 1fr);
+	  grid-template-rows: 1fr 1fr 1fr;
+	background: rgba(14, 6, 48, 0.21);
+	  grid-auto-columns: 4;
+	  grid-column-gap: 20px;
+	  grid-row-gap: 20px;
+	  grid-auto-rows: auto;
+	/* border: solid 2px yellow; */
+	place-items: center;
+  }
 }
+
+@media screen and (min-width: 1800px) {
+	.MoviesContainer {
+		overflow-y: scroll;
+		display: grid;
+		  grid-template-columns: repeat(5, 1fr);
+		  grid-template-rows: 1fr 1fr 1fr;
+		background: rgba(14, 6, 48, 0.21);
+		  grid-auto-columns: 4;
+		  grid-column-gap: 20px;
+		  grid-row-gap: 20px;
+		  grid-auto-rows: auto;
+		/* border: solid 2px yellow; */
+		place-items: center;
+	  }
+	}
```

Comment:
Nice use of responsive design principles here- our movie posters look a lot more polished now.